src/platform/graphics/constants.js

```diff
@@ -655,12 +782,36 @@ export const pixelFormatInfo = new Map([
     [PIXELFORMAT_PVRTC_4BPP_RGBA_1, { name: 'PVRTC_4BPP_RGBA_1', blockSize: 8 }],
     [PIXELFORMAT_ASTC_4x4, { name: 'ASTC_4x4', blockSize: 16 }],
     [PIXELFORMAT_ATC_RGB, { name: 'ATC_RGB', blockSize: 8 }],
-    [PIXELFORMAT_ATC_RGBA, { name: 'ATC_RGBA', blockSize: 16 }]
+    [PIXELFORMAT_ATC_RGBA, { name: 'ATC_RGBA', blockSize: 16 }],
+
+    // uncompressed integer formats (WebGL2 Only)
+    [PIXELFORMAT_R8I,       { name: 'R8I', size: 1, channelSize: 1 }],
+    [PIXELFORMAT_R8UI,       { name: 'R8UI', size: 1, channelSize: 1 }],
+    [PIXELFORMAT_R16I,       { name: 'R16I', size: 2, channelSize: 2 }],
+    [PIXELFORMAT_R16UI,       { name: 'R16UI', size: 2, channelSize: 2 }],
+    [PIXELFORMAT_R32I,       { name: 'R32I', size: 4, channelSize: 4 }],
+    [PIXELFORMAT_R32UI,       { name: 'R32UI', size: 4, channelSize: 4 }],
+    [PIXELFORMAT_RG8I,       { name: 'RG8I', size: 2, channelSize: 1 }],
+    [PIXELFORMAT_RG8UI,       { name: 'RG8UI', size: 2, channelSize: 1 }],
+    [PIXELFORMAT_RG16I,       { name: 'RG16I', size: 4, channelSize: 2 }],
+    [PIXELFORMAT_RG16UI,       { name: 'RG16UI', size: 4, channelSize: 2 }],
+    [PIXELFORMAT_RG32I,       { name: 'RG32I', size: 8, channelSize: 4 }],
+    [PIXELFORMAT_RG32UI,       { name: 'RG32UI', size: 8, channelSize: 4 }],
+    [PIXELFORMAT_RGBA8I,       { name: 'RGBA8I', size: 4, channelSize: 1 }],
+    [PIXELFORMAT_RGBA8UI,       { name: 'RGBA8UI', size: 4, channelSize: 1 }],
+    [PIXELFORMAT_RGBA16I,       { name: 'RGBA16I', size: 8, channelSize: 2 }],
+    [PIXELFORMAT_RGBA16UI,       { name: 'RGBA16UI', size: 8, channelSize: 2 }],
+    [PIXELFORMAT_RGBA32I,       { name: 'RGBA32I', size: 16, channelSize: 4 }],
+    [PIXELFORMAT_RGBA32UI,       { name: 'RGBA32UI', size: 16, channelSize: 4 }]
```

Comment:
Seems some superfluous spaces have been inserted here. Can you remove them, please?